README.md

```diff
@@ -1,378 +1,353 @@
+
+7. Save your changes
+  
+#### Add the bot to your server <a name="add-the-bot-to-your-server"></a>
+
+1. Still in your bot's application, click on `OAuth2`
+
+2. Enable the following scopes
+
+-  `bot`
+
+-  `applications.commands`
+
+3. Enable the Administrator permission
+
+- Note: this is likely to change in the future depending on how much access we want the bot to have
+
+4. Copy the generated link and paste it in a browser
+
+5. Add the bot to your testing server
 
- - [Youtube Tutorials](https://www.youtube.com/watch?v=KZ3tIGHU314&list=PLpmb-7WxPhe0ZVpH9pxT5MtC4heqej8Es) : These were the ones we watched. Highly recommend watching and following along.
- - [Discord.js Documentation](https://discord.js.org/)
- - [Discord Developer Portal](https://discord.com/developers/applications)
+  
+
+### Project Installation <a name="project-installation"></a>
+
+  
+
+#### Node Dependencies <a name="node-dependencies"></a>
+
+This project uses Node.js with the following dependencies and versions:
 
-# Required Packages and Version
+  
 
 ```
+
+"dayjs": "^1.11.11",
+
 "discord.js": "^14.15.2",
+
 "dotenv": "^16.4.5",
+
+"ms": "^2.1.3",
+
 "nodemon": "^3.1.0"
+
 ```
-**Important** : Make sure you have a `.env` set up for the project. An example is also provided in the repository. Here is an example:
 
-    DISCORD_TOKEN='example bot token here'
-Generate a token for the application by going to the wanted application under the [discord developer portal](https://discord.com/developers/applications) then going to the Bot tab.
+  
+
+#### .env Set Up <a name="env-set-up"></a>
+
+1. Create a duplicate of `example.env` and rename it to `.env`
+
+2. Open `.env` and replace the template data
+
+-  `DISCORD_TOKEN` is your bot's token. This can be found in your application under bot. Click `Reset Token` to show it.
+
+-  `TESTSERVER_ID` is your server's id, right click on your server's name in the top left, and click `Copy Server ID`
+
+-  `DEV_IDS` - is an array of user id's. These are used to restrict who has access to certain commands of the bot. To get your user ID, right click on your account in bottom left icon, click `Copy User ID`
+
+3. Verify that all of these ids/tokens are strings.
+
+  
 
-# Setup
+Note: anything related to ids should be added to `.env` for privacy
+
+  
+
+#### Lint Set Up <a name="lint-set-up"></a>
+
+With the addition of [Git hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) to our project, developers can automate the
 
-With the addition of [Git hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) to our project, developers can automate the 
 linting and formatting process. Run the following command to set up git hooks for your local repository. Go to the [Linting Section](https://github.com/RIT-Crowd-Comic/Bot?tab=readme-ov-file#Linting) to read more about configuring ESLint and Git hooks.
```

Comment:
Either remove the sentence containing a link to the linting section or yoink the entire Linting section from main (in which the link probably doesn't need to be updated)